### PR TITLE
Towards #3562 - Readiness Checks formats

### PR DESCRIPTION
**Pull request by gkleiman**

### CODE REVIEW

File: src/main/scala/mesosphere/marathon/core/readiness/ReadinessCheckResult.scala [hunk 1]
```diff
@@ -7,5 +7,4 @@ case class ReadinessCheckResult(name: String,
                                 ready: Boolean,
                                 httpResponse: Option[HttpResponse])
```
Comment (kolloch) [inline src/main/scala/mesosphere/marathon/core/readiness/ReadinessCheckResult.scala]: I don't see why we shouldn't use the same name as in the JSON here. "lastResponse"

Comment (aquamatthias) [inline src/main/scala/mesosphere/marathon/core/readiness/ReadinessCheckResult.scala]: And if we rename this, do we need the explicit Format? 
Can't we just use `Json.format[ReadnissCheckResult]`?

**Comment by gkleiman** [comment]
Updated the PR. The tests do pass locally, but both TeamCity and Travis report build errors, so there must be something wrong. I will look into this tomorrow, but hints are more than appreciated.

cc/ @kolloch

**Comment by aquamatthias** [comment]
Thanks.

**Pull request merged**